### PR TITLE
Add Game Boy Development

### PR DESCRIPTION
**Pull request by avivace**
A complete and curated list about the original 1989 Game Boy handheld console.
Everything available to start studying the hardware/software mechanism, create a working ROM, an emulator or simply play with the console architecture.

---

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please don't remove the pull request template. You're supposed to fill it out and have it completed before submitting a pull request. I've added it back. Please fill it out.

**Comment by sindresorhus** [comment]
Cool idea!

**Comment by avivace** [comment]
Ups, done!

File: readme.md [hunk 1]
```diff
@@ -282,6 +282,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Chess](https://github.com/hkirat/awesome-chess)
 - [LÖVE](https://github.com/love2d-community/awesome-love2d) - Game engine.
 - [PICO-8](https://github.com/felipebueno/awesome-PICO-8) - Fantasy console.
+- [GBDev](https://github.com/avivace/awesome-gbdev) - Original Game Boy
```
Comment (sindresorhus) [inline readme.md]: GBDev => Game Boy Development

And drop the description

Comment (avivace) [inline readme.md]: Makes sense. Thank you.

**Comment by sindresorhus** [comment]
Feedback:

- Readme title: `awesome-gbdev` => `Awesome Game Boy Development`
- Link and description should be separated with a dash `-`. See how it's done in other lists.
- Inconsistent use of Game Boy. GameBoy and Gameboy is incorrect.
- `asm` and `gbdk` under `Programming Tutorials` should be sub-headings, not plain text.
- Use the SVG version of the "Public Domain" license badge.

> only has awesome items **with descriptions**   

Not followed

> has been checked for inconsistent formatting, spelling, and grammar (**This includes starting each link description with an uppercase character and ending it with a period ., and consistent naming**),

Not followed

> has contribution guidelines,

This should be a `contributing.md` file. See this and other lists for example.

**Comment by avivace** [comment]
Gotcha, should be a lot better now.

**Pull request merged**